pkg/client/fieldmanager_test.go

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
@ahmetb Can you please adjust the filename to fieldowner_test.go? (just for main is okay)

Reply:
sent #2777